File: app/src/main/java/seigneur/gauvain/mycourt/ui/shots/ShotsFragmentModule.java
package seigneur.gauvain.mycourt.ui.shots;

import android.support.v4.app.Fragment;

import dagger.Binds;
import dagger.Module;
import seigneur.gauvain.mycourt.di.scope.PerFragment;

@Module()
public abstract class ShotsFragmentModule {
    @Binds
    @PerFragment
    abstract Fragment fragment(ShotsFragment shotsFragment);

}

File: app/src/main/java/seigneur/gauvain/mycourt/ui/shots/ShotsFragmentModule.kt
package seigneur.gauvain.mycourt.ui.shots

import androidx.fragment.app.Fragment

import dagger.Binds
import dagger.Module
import seigneur.gauvain.mycourt.di.scope.PerFragment

@Module
abstract class ShotsFragmentModule {
    @Binds
    @PerFragment
    internal abstract fun fragment(shotsFragment: ShotsFragment): androidx.fragment.app.Fragment

}